From c21a3616bacfa8d7d56f0a033ceef3194a563908 Mon Sep 17 00:00:00 2001
From: Fernando Reyna <fernando.m.reyna.diaz@intel.com>
Date: Wed, 17 Feb 2016 12:25:46 -0600
Subject: [PATCH 4/6] OV2680: Changed api_device_id.

CHT-T3 MRD is looking for OVTI2680 but the api_device_id
is using XXOV2685 so it was changed to work on MRD.

Change-Id: I865057435ad1799c9d6bdeac7cd8bdef3b807cf1
Tracked-On: https://jira01.devtools.intel.com/browse/OAM-20600
Signed-off-by: Fernando Reyna <fernando.m.reyna.diaz@intel.com>
Signed-off-by: Marco Salgado <marco.antonio.salgado.martinez@intel.com>
Signed-off-by: Lopez Esparza, Ricardo <ricardo.lopez.esparza@intel.com>
Reviewed-on: https://android.intel.com:443/471770
---
 drivers/media/i2c/atomisp/ov2680.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/drivers/media/i2c/atomisp/ov2680.c b/drivers/media/i2c/atomisp/ov2680.c
index aec55c2..0248577 100644
--- a/drivers/media/i2c/atomisp/ov2680.c
+++ b/drivers/media/i2c/atomisp/ov2680.c
@@ -1642,6 +1642,7 @@ out_free:
 
 static struct acpi_device_id ov2680_acpi_match[] = {
 	{"XXOV2680"},
+	{"OVTI2680"},
 	{},
 };
 MODULE_DEVICE_TABLE(acpi, ov2680_acpi_match);
-- 
1.9.1

